storybookjs/storybook
# fix(preview): zoom iframe content instead zoom iframe wrapper

Issue: #5812

## What I did
If zoom in iframe wrapper then will zoomed scrollbars. For fix it i change scalable component: preview iframe body instead iframe wrapper.

## How to test
In [issue](https://github.com/storybooks/storybook/issues/5812) you can find detailed instructions.

- Is this testable with Jest or Chromatic screenshots?
yes, i updated jest snapshots
- Does this need a new example in the kitchen sink apps?
no
- Does this need an update to the documentation?
no

If your answer is yes to any of these, please make sure to include it in your PR.
